let num1 = 5;
let num2 = "hello"; 
try { 
let result = num1 + num2; 
console.log(result);
 } catch(error) { 
console.log("Oops, something went wrong! Error message:", error); 
}